packages/astro/src/container/pipeline.ts

```diff
@@ -22,17 +22,19 @@ export class ContainerPipeline extends Pipeline {
 	static create({
 		logger,
 		manifest,
+		astroConfig,
 		renderers,
 		resolve,
 		serverLike,
 		streaming,
 	}: Pick<
 		ContainerPipeline,
-		'logger' | 'manifest' | 'renderers' | 'resolve' | 'serverLike' | 'streaming'
+		'logger' | 'manifest' | 'astroConfig' | 'renderers' | 'resolve' | 'serverLike' | 'streaming'
```

Comment:
Unfortunately, this change can't be accepted because it won't work in SSR (on-demand pages).

The Astro configuration disappears after the build, meaning it will break.

Instead, internally, we have a serialised version of the configuration called manifest. This can be accessed via `pipeline.manifest`.

In `dev`, we have function called `createDevelopmentManifest`, which contains the field base.